### ISSUE TITLE
Incompatible with XR18

### ISSUE THREAD

**Issue by benbogart** [essej/sonobus#2]
Hello!  This is a fantastic project.  I am a regular JackTrip user and looking forward to see what SonoBus can do (especially with compression!)

I have an Behringer XR18 mixer that I use with JackTrip.  https://www.behringer.com/product.html?modelCode=P0BI8. SonoBus will not connect to it.  I can connect to my internal sound card and even send sound through loopback to the mixer successfully, but I cannot send or receive sound directly to or from the mixer from SonoBus.

What data can I share to help figure out what is happening?

**Comment by essej** [comment]
What platform are you on (mac, windows, or linux)? Can you say exactly what happens when you select it in the audio setup?

**Comment by benbogart** [comment]
I tried again today to provide more detailed feedback and everything is working as expected now.  I don't know what changed except possibly a system restart.

For reference if this comes up again.  I am on Mac OSX 10.14.6.  There was no signal to or from the mixer when I tried to connect to it from sonobus.  I could interact with the mixer with other programs, and even by sending signal from sono bus to the mixer through virtual inputs.

In anycase, it seems to be working now.  I'm looking forward to testing it out!
